Ajout tâche sur des conditions imbriquées

FrancoisDuchene opened the pull request:
Ajout de deux tâches blockly sur les conditions.
* Recrée : concept de conditions imbriquées (un if else dans un if)
* Points d'interro : concept sur les conditions d'une manière générale

FrancoisDuchene committed:
Ajout tache Recree sur des conditions imbriquées

FrancoisDuchene committed:
ajout tache sur conditions

FrancoisDuchene committed:
ajout commentaires

AlexandreDubray commented:
Contextualisation super sympa et les feedback sont bien expliqué! Pour la tâche point interro, peut-être spécifier qu'il faut dire que maman est contente si jamais Michel a entre 10 et 16? Et une type pour "Michel ne note qu'entre 0 et 20"? Sinon on pourra merger quand tu auras appliquer la license OS.

FrancoisDuchene committed:
Ajouté mentions à license utilisée + redéfinition contexte "points-interro"

La license utilisée est celle d'Inginious
Redéfini le contexte "points-interro" pour être plus clair
ajouts de commentaires dans le code

FrancoisDuchene commented:
J'ai ajouté des mentions à la licence utilisée (celle d'Inginious) et j'ai rendu le contexte de "points_interro" plus clair.

AlexandreDubray commented:
Vous devez mettre un header AGPL, votre code ici ne fait pas partie d'INGInious mais est un exercice qu'on peut utilisé via INGInious.

FrancoisDuchene committed:
header AGPL-3

FrancoisDuchene commented:
J'ai ajouté les headers qui se trouvent dans à la fin du fichier de license sur le repo, j'espère que c'est suffisant

FrancoisDuchene committed:
correction tests Recree

FrancoisDuchene committed:
correction provisoire erreur blockly (voir FIXME)
